lib/muffin/base.rb

```diff
@@ -19,5 +21,15 @@ def initialize(request: nil, params: nil, scope: nil)
 
       assign_attributes
     end
+
+    def persisted?
+      attributes[:id].present?
+    end
+
+    # fields_for checks wether an object responds to [foo_attributes=]
+    def respond_to_missing?(method_name, include_private = false)
+      return true if method_name.to_s[/_attributes=\Z/]
+      super
+    end
```

Comment:
I think this should to to `Attributes` as well (imagine a 2-level-nested form, e.g. the `ff.fields_for :tags` case from #3.